cgi-bin/DW/Controller/Entry.pm

```diff
@@ -344,8 +345,10 @@ sub _init {
         $panels = $u->entryform_panels;
         $formwidth = $u->entryform_width;
         $min_animation = $u->prop( "js_animations_minimal" ) ? 1 : 0;
+        $displaydate_check = $u->displaydate_check ? 1 : 0;
     } else {
         $panels = LJ::User::default_entryform_panels( anonymous => 1 );
+        $displaydate_check = 0;
```

Comment:
This doesn't need to be set explicitly, btw, because it defaults to 0 when you declare $displaydate_check above.